### DISCUSSION TITLE
关于泡泡dns 默认禁用HTTPS类型

### DISCUSSION THREAD

**Discussion by biliblihuorong** [kkkgo/PaoPaoDNS#210] [问答和帮助]
这几天cloud flare重新开始测试ech，而我发现泡泡dns请求时没有ech值，随后我查到mosdns配置中有明确禁用HTTPS类型，我现在把他注释，会对泡泡dns 有什么影响吗？以及为什么泡泡dns 会默认禁用HHTPS类型

**Comment by kkkgo** [comment]
参考 https://www.v2ex.com/t/712074
结论：会导致绕过DNS提供的A/AAAA记录，导致分流失效。
如果你有某些域名需要利用这个类型，可以使用custom_mod功能绕过。

**Reply by jaesony** [reply]
我说ech测试怎么有问题，原来是这样，删了禁用类型64/65那两行就可以了。我用的ospf分流，应该没影响吧

**Comment by biliblihuorong** [comment]
谢谢大佬
